services/direct_delivery/direct_delivery_example.py

```diff
@@ -0,0 +1,31 @@
+import smtplib
+import email.utils
+from email.mime.text import MIMEText
+
+SMTP_SERVER = 'in.mailsac.com'
+SMTP_PORT = 587
+FROM_ADDRESS = 'no-reply@mywebapp.com'
+FROM_NAME = 'Mywebapp Customer Support'
+TO_ADDRESS = 'mycustomer@gmail.com'
+SUBJECT = 'Password reset'
+BODY_TEXT = ('Click on the link to reset you password\r\n'
+             'https://mywebapp.com/password-reset/PasswordResetToken'
+            )
+
+msg = MIMEText(BODY_TEXT)
+msg['Subject'] = SUBJECT
+msg['From'] = email.utils.formataddr((FROM_NAME, FROM_ADDRESS))
+msg['To'] = TO_ADDRESS
+
+conn = smtplib.SMTP(SMTP_SERVER, SMTP_PORT)
+conn.set_debuglevel(True)
+conn.ehlo()
+conn.starttls()
+conn.ehlo()
+try:
+    conn.sendmail(FROM_ADDRESS, TO_ADDRESS, msg.as_string())
+    conn.close()
+except Exception as e:
+    print('Error: ', e)
+else:
+    print('Email Sent')
```

Comment:
Maybe add sample output results. I find this pretty helpful often as a developer. Not required though.

```python
send: 'ehlo 131.50.168.192.in-addr.arpa\r\n'
reply: '250-smtp-in2-172-31-42-57 Welcome! [172.31.42.57:25]\r\n'
reply: '250-SIZE 1048576\r\n'
reply: '250-STARTTLS\r\n'
reply: '250-PROXY PROXY\r\n'
reply: '250 HELP\r\n'
reply: retcode (250); Msg: smtp-in2-172-31-42-57 Welcome! [172.31.42.57:25]
SIZE 1048576
STARTTLS
PROXY PROXY
HELP
send: 'STARTTLS\r\n'
reply: '220 Ready to start TLS\r\n'
reply: retcode (220); Msg: Ready to start TLS
send: 'ehlo 131.50.168.192.in-addr.arpa\r\n'
reply: '250-smtp-in2-172-31-42-57 Welcome! [172.31.42.57:25]\r\n'
reply: '250-SIZE 1048576\r\n'
reply: '250-PROXY PROXY\r\n'
reply: '250 HELP\r\n'
reply: retcode (250); Msg: smtp-in2-172-31-42-57 Welcome! [172.31.42.57:25]
SIZE 1048576
PROXY PROXY
HELP
send: 'mail FROM:<no-reply@mywebapp.com> size=296\r\n'
reply: '250 Accepted\r\n'
reply: retcode (250); Msg: Accepted
send: 'rcpt TO:<mycustomer@gmail.com>\r\n'
reply: '250 Accepted\r\n'
reply: retcode (250); Msg: Accepted
send: 'data\r\n'
reply: '354 Enter message, ending with "." on a line by itself\r\n'
reply: retcode (354); Msg: Enter message, ending with "." on a line by itself
data: (354, 'Enter message, ending with "." on a line by itself')
send: 'Content-Type: text/plain; charset="us-ascii"\r\nMIME-Version: 1.0\r\nContent-Transfer-Encoding: 7bit\r\nSubject: Password reset\r\nFrom: Mywebapp Customer Support <no-reply@mywebapp.com>\r\nTo: mycustomer@gmail.com\r\n\r\nClick on the link to reset you password\r\nhttps://mywebapp.com/password-reset/PasswordResetToken\r\n.\r\n'
reply: '250 OK : queued as 9702DoPYyXnTaIEBP0AOl1AE\r\n'
reply: retcode (250); Msg: OK : queued as 9702DoPYyXnTaIEBP0AOl1AE
data: (250, 'OK : queued as 9702DoPYyXnTaIEBP0AOl1AE')
Email Sent
```

Reply:
Added console debug output to the example code.